### PR TITLE
Edits to session 3

### PR DESCRIPTION
**Pull request by valyo**

### CODE REVIEW

File: docs/3_project_life_cycle_session.md [hunk 1]
```diff
@@ -61,8 +61,7 @@ A project can be deleted in cases of incorrect project information or errors, an
 
 Projects are **Available** for a certain (chosen by the units) period of time, Days in Available (DiA). After that their status automatically changes to **Expired**. Data cannot be changed in this state, but the project can be re-released up to *two times*. After two re-releases, a project can only be archived.
 
-ToDO add task for checking project status history and note deadline
-- [ ] TASK: 
+- [ ] TASK: Check the current project status and the status history. Observe the deadline date for the current status.
```
Comment (aishling-scilifelab) [inline docs/3_project_life_cycle_session.md:64]: _Observe the deadline date for the current status._

This part is slightly confusing to me - but does it mean the deadline date for whatever the current status is (but isn't it always In Progress at this point if users are following the steps)?

Comment (rv0lt) [reply docs/3_project_life_cycle_session.md:64]: At that point, the project should be expired, no?

Comment (valyo) [reply docs/3_project_life_cycle_session.md:64]: yes, what ever status (In Progress or Available) the deadline is printed in the output

Comment (aishling-scilifelab) [reply docs/3_project_life_cycle_session.md:64]: Cool, I understand.

**Review by aishling-scilifelab — APPROVED** [review]

File: docs/3_project_life_cycle_session.md (hunk 1 above)
Comment (valyo) [reply docs/3_project_life_cycle_session.md:64]: > At that point, the project should be expired, no?

No, how could they expire so soon

Comment (rv0lt) [reply docs/3_project_life_cycle_session.md:64]: my bad. I got confused and thought there was a task about expiring a project

**Review by rv0lt — APPROVED** [review]

**Pull request merged**